refinery/analysis_manager/tasks.py

```diff
@@ -14,7 +14,6 @@
 import requests
```

Comment:
`dataset_manager.views:90`
`  full_isa_tab_url = get_full_url(request.POST['isa_tab_url'])` -> `  full_isa_tab_url = request.POST['isa_tab_url']`

Reply:
I'm going to put a TODO on this line: No harm done by having an extra get_full_url call, but if it did confuse something, it could be hard to track down.